Update adding-model.md

Rick-Anderson opened the pull request:
@tdykstra  alot of customers report PMC not working and console is working.

tdykstra commented on aspnetcore/tutorials/first-mvc-app/adding-model.md:
````diff
@@ -102,6 +102,18 @@ The `Add-Migration` command creates code to create the initial database schema.
+* Add [EF Core tooling](xref:data/ef-mvc/migrations#entity-framework-core-nuget-packages-for-migrations) to the *.csproj* file.
+* Run the following commands from the console (in the project directory):
+
+  ```console
````
to put this under the Run the following commands bullet, the opening ``` is indented 2, but following lines and closing ``` are indented more

tdykstra commented on aspnetcore/tutorials/first-mvc-app/adding-model.md:
```diff
@@ -102,6 +102,18 @@ The `Add-Migration` command creates code to create the initial database schema.
 
 The `Update-Database` command runs the `Up` method in the *Migrations/\<time-stamp>_InitialCreate.cs* file, which creates the database.
 
+You can perform the preceeding steps using the command-line interface (CLI) rather than the PMC:
```
This is kind of lost without special formatting like a Note.  Also might consider noting this earlier, or people could get frustrated doing the instructions and not realize there's an alternative.

Rick-Anderson commented:
@tdykstra  I'm wondering if we should add the following after the CLI instructions.
Please leave a note if the PMC steps failed.

Addressed your comments.

tdykstra commented:
Or give them an open-new-issue link if you think it's a recurring problem that we haven't figured out the cause for yet.

Rick-Anderson commented:
>Or give them an open-new-issue link if you think it's a recurring problem that we haven't figured out the cause for yet.

We'll get dozens of pilot error issues. I'm hoping my changes will help as I've gotten alot of comments saying it doesn't work. SOme of them were running Windows 7.
